### DISCUSSION TITLE
Option de rappel pour les gestionnaires pour renouveler les infos pour les convives

### DISCUSSION THREAD

**Discussion by hfroot** [betagouv/ma-cantine#1093] [Idées]
Est-ce que on veut donner les gestionnaires l'option de demander un rappel quand iels doivent renouveler l'affiche dans la cantine ?

En plus, car l'affiche doit être mettre à jour au moins une fois par an, est-ce qu'on veut ajouter la date générer sur l'affiche ?

**Comment by valeriemerle** [comment]
la date est déjà sur l'affiche puisque c'est l'année -1. Le rappel pourrait être une bonne idée et permettrait aussi la mise à jour des données pour ceux qui ne l'aurait pas fait

**Reply by hfroot** [reply]
C'est vrai - alors est-ce que on veut supposer que toutes les cantines vont renouveler l'affiche au début de chaque année ? Si c'est le cas peut-être une solution avec le newsletter est mieux.
